contrib/inventory/rackhd.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python
+import json
+import requests
+import os
+import argparse
+import types
+
+RACKHD_URL = 'http://localhost:8080'
+
+class RackhdInventory(object):
+    def __init__(self, nodeids):
+        self._inventory = {}
+        for nodeid in nodeids:
+            self._load_inventory_data(nodeid)
+        output = '{\n'
+        for nodeid,info in self._inventory.iteritems():
+            output += self._format_output(nodeid, info)
+            output += ',\n'
+        output = output[:-2]
+        output += '}\n'
```

Comment:
Looks like instead of creating a full python data structure, this code is trying to hand craft JSON via string concatenation.  See my other comment about just using a data structure and using `json.dumps` at the very end.